podman/domain/registry_data.py

```diff
@@ -63,7 +68,7 @@ def has_platform(self, platform: Union[str, Mapping[str, Any]]) -> bool:
 
         if isinstance(platform, dict):
             if not {"os", "architecture"} <= platform.keys():
-                version = self.client.version()
+                version = self.client.version()  # type: ignore[operator]
```

Comment:
Here `self.client` is a `APIClient` where `.version` is a string. It would be in `PodmanClient` that `.version` would be callable. This specific line should throw an error as it is.